common/changes/@gooddata/sdk-ui-all/ine-tnt-1407_2023-03-17-15-15.json

```diff
@@ -0,0 +1,10 @@
+{
+    "changes": [
+        {
+            "packageName": "@gooddata/sdk-ui-all",
+            "comment": "Single selection variant of Attribute Filter component",
+            "type": "none"
+        }
+    ],
+    "packageName": "@gooddata/sdk-ui-all"
+}
```

Comment:
I am missing basic e2e for new Attribute filer mode in sdk-ui-tests-e2e (Cypress).

Happy path, Parent filter relation, Mutli vs Single select conversion.

Reply:
Will be delivered by follow up PR